web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -39,8 +40,8 @@ const BookingModalForm = props => {
       return [
         {
           label: `Request ${
-            bookingDuration === 0.5 ? 'Half' : bookingDuration
-          } ${bookingDuration > 1 ? 'Days' : 'Day'}`,
+            eventTypeId !== eventTypes.ANNUAL_LEAVE ? '' : bookingDuration === 0.5 ? 'Half' : bookingDuration}
+          ${eventTypeId !== eventTypes.ANNUAL_LEAVE ? 'WFH' : bookingDuration > 1 ? 'Days' : 'Day'}`,
           event: createEvent,
```

Comment:
A small thing but might be better to abstract this ternary string generation to it's own function